nservicebus/sqlserver/index.md

```diff
@@ -28,38 +28,63 @@ The SQL Server transport is a hybrid queueing system which is neither store-and-
 
-The SQL Server transport is a good choice for a pilot project to prove feasibility of NServiceBus in a given organization. It does not require anything more than a database. Such a project would usually be small so it is not likely to use more than one physical database in which case no DTC is required.
+The queue tables can be hosted in the same SQL Server catalog as business and persistence data. NServiceBus endpoint can be hosted in the ASP.NET worker process. In some cases there might be a need for a separate process for hosting the NServiceBus endpoint (due to security considerations or IIS worker process life-cycle management). Because system consists of a single logical service or bounded context, there is usually no need to create separate schema for the queues.
 
+### Small
 
-### Messaging framework for an application using SQL Azure
+The SQL Server transport is a good choice for a pilot project to prove feasibility of NServiceBus in a given organization as well as for a small, well-defined green field application. It usually requires nothing more than a single shared SQL Server instance.
 
-The SQL Azure's limitations include no support for distributed transactions and relatively small database size. This makes it an ideal candidate for a [multi-database mode](deployment-options.md) SQL Server transport with [Outbox](/nservicebus/outbox/). Each endpoint could have its own database that meets the size limit. Outbox would guarantee *exactly-once* processing of messages.
\ No newline at end of file
+The best option is to store the queues in the same catalog as the business data. Schemas can be used to make maintenance easier. 
+
+### Medium to large
+
+For larger systems it usually makes more sense to have dedicated catalogs for each service or bounded context. NoSQL data stores can be used in some services, depending on data access requirements. The SQL catalogs might be hosted in a single instance of SQL Server or in separate instances depending on the IT policy.
+
+The best option is to have a dedicated catalog for the queues. This approach allows to use different scaling up strategies for the queue catalog. It also allows to use a dedicated back up policy for the queues (because data in queues are much more ephemeral). 
+
+Is such case local transactions are not enough to ensure `exactly-once` message processing. When required, one option is to use Microsoft Distributed Transaction Coordinator and distributed transactions (NServiceBus with SQL Server transport is DTC-enabled by default). In this mode the message receive transaction and all data modifications that result from processing the message are part of one distributed transaction that spans two SQL catalogs (possibly stored on two different instances).
```

Comment:
> Is such case local transactions are not enough to ensure `exactly-once` message processing.

Didn't you just tell me to keep all queues in one catalog, how did we end up here?!